lib/utils/interfaces/auth.interface.ts

```
@@ -1,7 +1,65 @@
+  type: string;
+  name: string;
+  notificationToken: string;
+  isActive: boolean;
+}
+
+export interface IAuthContextProps {
+  authToken: string;
+  setAuthToken: Dispatch<SetStateAction<string>>;
+  user: IUser | null;
+  setUser: Dispatch<SetStateAction<IUser | null>>;
+  checkEmail: (email: string) => Promise<boolean | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)